recipes/mitos/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "mitos" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: de00eecb47f5a67c4a27104c87b4d6e26f1c9eddc86369864f7aed1b486c21c6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - biopython ==1.73
+    - ete2 ==2.3.10
+    - pip
+    - python <3
+    - reportlab
+  run:
+    - biopython ==1.73
+    - ete2 ==2.3.10
+    - python <3
+    - reportlab
+    - blast-legacy ==2.2.26
+    - infernal ==1.0.2
+    - viennarna <2
+    - r-base  
+    - r-ggplot2 ==3.1.1
+    - openjdk
+
+test:
+  imports:
+    - mitos
+    - mitos.align
+    - mitos.bedfile
+    - mitos.blast
+    - mitos.codon
+    - mitos.draw
+    - mitos.extprog
+    - mitos.feature
+    - mitos.gb
+    - mitos.gfffile
+    - mitos.mergefeature
+    - mitos.mitfi
+    - mitos.mito
+    - mitos.mitofile
+    - mitos.msa
+    - mitos.rEvoluzer
+    - mitos.rna
+    - mitos.sequence
+    - mitos.tax
+    - mitos.tbl
+    - mitos.trna
+    - mitos.update
+    - mitos.webserver
+  commands:
+    - runmitos.py -h
+
+about:
+  home: http://mitos.bioinf.uni-leipzig.de
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE 
+  summary: MITOS
+  dev_url: https://gitlab.com/Bernt/MITOS 
+
+extra:
+  recipe-maintainers:
+    - Bernt
```

Comment:
that seems to be incorrect :)

Reply:
Just took the gitlab user name. Changed it to my full name.